cpp/src/arrow/util/cpu_info.cc

```diff
@@ -363,6 +363,10 @@ int64_t LinuxParseCpuFlags(const std::string& values) {
     {"avx512bw", CpuInfo::AVX512BW},
     {"bmi1", CpuInfo::BMI1},
     {"bmi2", CpuInfo::BMI2},
+    {"avx512ifma", CpuInfo::AVX512IFMA},
+    {"avx512vbmi", CpuInfo::AVX512VBMI},
+    {"avx512_vbmi2", CpuInfo::AVX512VBMI2},
+    {"avx512_vnni", CpuInfo::AVX512VNNI},
```

Comment:
```suggestion
    {"avx512vbmi2", CpuInfo::AVX512VBMI2},
    {"avx512vnni", CpuInfo::AVX512VNNI},
```

Reply:
@kou Thanks for your suggestions! But I think "avx512_vbmi2"/"avx512_vnni" could not be renamed because they have to be consistent with the value of flags in the /proc/cpuinfo file.